src/mcmcchains.jl

```diff
@@ -61,8 +61,10 @@ attributes_dict(::Nothing) = Dict()
 
 function section_dict(chns::Chains, section)
     ndraws, _, nchains = size(chns)
-    loc_names = string.(getfield(chns.name_map, section))
-    vars_to_locs = varnames_locs_dict(loc_names)
+    loc_names_old = getfield(chns.name_map, section) # old may be Symbol or String
+    loc_names = string.(loc_names_old)
+    loc_str_to_old = Dict(name_str => name_old for (name_str, name_old) in zip(loc_names, loc_names_old))
```

Comment:
Suggestion from the new format checking action (https://github.com/arviz-devs/ArviZ.jl/pull/77/checks?check_run_id=1076393897#step:5:29)
```suggestion
    loc_str_to_old =
        Dict(name_str => name_old for (name_str, name_old) in zip(loc_names, loc_names_old))
```